v1/css.yaml

```diff
@@ -0,0 +1,96 @@
+swagger: '2.0'
+info:
+  version: '1.0.0-beta'
+  title: CSS for mobile apps
+  description: API for retrieving CSS for mobile apps
+  termsOfService: https://www.mediawiki.org/wiki/REST_API#Terms_and_conditions
+  contact:
+    name: Reading Infrastructure
+    url: https://www.mediawiki.org/wiki/Wikimedia_Reading_Infrastructure_team
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
+  /css/mobile/base:
+    get:
+      tags:
+        - Mobile
+      summary: Get base CSS for mobile apps.
+      description: |
+        Gets common CSS for mobile apps need to properly display pages using Page Content Service.
+
+        Stability: [experimental](https://www.mediawiki.org/wiki/API_versioning#Experimental)
+      produces:
+        - application/json; charset=utf-8; profile="https://www.mediawiki.org/wiki/Specs/CSS/1.0.0"
+      responses:
+        '200':
+          description: Success
+          headers:
+            ETag:
+              description: Different values indicate that the content has changed
+        default:
+          description: Error
+          schema:
+            $ref: '#/definitions/problem'
+      x-request-handler:
+        - get_from_pcs:
+            request:
+              method: get
+              headers:
+                cache-control: '{{cache-control}}'
```

Comment:
Why do we need to pass the `cache-control` header?

Reply:
@Pchelolo I want to use the cache-control values provided by PCS.

Reply:
@berndsi sure, but this is the `request` section, why would we forward client-specified `cache-control` to PCS?

Reply:
Ah, got it. I think I copied this from the other PCS endpoints. May need to look there as well.